Makefile

```diff
@@ -36,16 +37,23 @@ html:
 pdf:
 	make -C doc latexpdf
 
+PYENV_DIR=.virtualenv
 FORKED_FORMULAS_DIR=formulas
-FORMULAS=`python3 -c 'import sys; sys.path.append("scripts");from update_mrconfig import *; print(*get_org_repos(make_github_agent(), "salt-formulas"), sep="\n")'| egrep 'salt-formula-' | sed 's/salt-formula-//'`
+FORMULAS=`. $(PYENV_DIR)/bin/activate && python3 -c 'import sys; sys.path.append("scripts");from update_mrconfig import *; print(*get_org_repos(make_github_agent(), "salt-formulas"), sep="\n")'| egrep 'salt-formula-' | sed 's/salt-formula-//'`
```

Comment:
What if requirements are already installed on system and user does not need to care about virtualenv?

Reply:
there is a problem with pygithub. Documented.Fail to load it's MainClas, I found it to fail when installing systemwide/virtualenv with linking system libraries.

Reply:
pygithub package will be soon updated to latest version in Debian. Default should be to use system-wide stuff when available. Do not force user to use virtualenv.